src/it/dockerfile-docker-native-oracle-function/Dockerfile

```diff
@@ -7,9 +7,9 @@ RUN native-image --report-unsupported-elements-at-runtime -H:Class=com.fnproject
 
 FROM fnproject/fn-java-fdk:jre11-latest AS fnfdk
 
-FROM busybox:glibc
+FROM frolvlad/alpine-glibc:alpine-3.12
```

Comment:
@burtbeckwith if you want to build "mostly static" native images then besides adding the flag you already added in starter (https://github.com/micronaut-projects/micronaut-starter/pull/895/files) then you should use another Docker image like `distroless`. See https://gitlab.com/micronaut-projects/micronaut-graal-tests/-/blob/3.0.x-stable/.gitlab-ci.yml#L1312 instead of one based on glibc